src/SFA.DAS.EmployerIncentives.Domain/ValueObjects/CollectionPeriod.cs

```diff
@@ -6,7 +6,7 @@ namespace SFA.DAS.EmployerIncentives.Domain.ValueObjects
 {
     public class CollectionPeriod : ValueObject
     {
-        public CollectionPeriod(byte periodNumber, byte calendarMonth, short calendarYear, DateTime openDate, DateTime censusDate, short academicYear, bool active)
+        public CollectionPeriod(byte periodNumber, byte calendarMonth, short calendarYear, DateTime openDate, DateTime censusDate, short academicYear, bool active, bool periodEndInProgress)
```

Comment:
I don't think PeriodEndInProgress should be on the constructor since you shouldn't be able to create one in progress.  Only the domain logic should be able to set to in progress or the data mapper when hydrating.

Reply:
The mapper uses this method when hydrating.